addons/pos_self_order/views/product_views.xml

```diff
@@ -19,17 +19,19 @@
         <field name="model">product.template</field>
         <field name="inherit_id" ref="point_of_sale.product_template_form_view"/>
         <field name="arch" type="xml">
-            <xpath expr="//group[@name='pos']" position="inside">
-                <field name="self_order_available" invisible="not available_in_pos"/>
-            </xpath>
-            <group name="upsell" position="after">
-                <group string="Product Description for Self Order" name="description_self_order">
-                    <field name="description_self_order"
-                           placeholder="Information about your product for Self Order and Kiosk"
-                           nolabel="1"
-                           colspan="2"/>
+            <page name="pos" position="inside">
+                <group name="pos" position="inside">
```

Comment:
This double "xpath" seems strange to me, does it really work as expected?
Feels like either the `<page` node could be removed, or the two could be grouped into a `<xpath expr="//page[@name='pos']//group[@name='pos']" position=""/>`